nemo/src/rule_model/error/validation_error.rs

```diff
@@ -167,9 +168,13 @@ pub enum ValidationErrorKind {
     #[assoc(code = 233)]
     ImportExportInvalidIri,
     /// Import/Export: invalid SPARQL query
-    #[error(r#"invalid Sparql query resulting in oxigraph-parser error: `{oxi_error}`"#)]
     #[assoc(code = 234)]
+    #[error(r#"invalid Sparql query resulting in oxigraph-parser error: `{oxi_error}`"#)]
```

Comment:
error messages shouldn't be unnecessarily verbose, this should be something like 
```rust
"invalid SPARQL query: `{oxi_error}`"
```